certification/formatters/formatters_test.go

```diff
@@ -0,0 +1,83 @@
+package formatters_test
+
+import (
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"github.com/redhat-openshift-ecosystem/openshift-preflight/certification/formatters"
+	"github.com/redhat-openshift-ecosystem/openshift-preflight/certification/runtime"
+	"github.com/redhat-openshift-ecosystem/openshift-preflight/cmd"
+)
+
+var _ = Describe("Formatters", func() {
+	Describe("When getting a new formatter for a configuration", func() {
+		Context("with a valid configuration", func() {
+			cfg := runtime.Config{
+				ResponseFormat: "json",
+			}
+
+			It("should return a formatter and no error", func() {
+				formatter, err := formatters.NewForConfig(cfg)
+				Expect(err).To(BeNil())
```

Comment:
It would be great to replace this line with `Expect(err).ToNot(HaveOccurred())` for the sake of consistency.

Reply:
@samira-barouti Definitely; my apologies. Thanks for catching this!

Reply:
This is done.